project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -40,34 +40,49 @@
 @RunWith(JUnit4.class)
 public final class PlacesFetcherTest {
 
-  private static final String PLACE_URL = "https://www.google.com/"; // used for Places
-  private static final URL PLACES_DETAILS_URL = createTestURL(PLACE_URL); // used for PlaceDetails
+  private static final String PLACE_WEBSITE = "https://www.google.com/"; // used for Places
+  private static final String PLACE_GOOGLE_URL = "https://www.google.com/"; // used for Places
+  private static final URL PLACE_DETAILS_WEBSITE =
+      createTestURL(PLACE_WEBSITE); // used for PlaceDetails
+  private static final URL PLACE_DETAILS_GOOGLE_URL =
+      createTestURL(PLACE_GOOGLE_URL); // used for PlaceDetails
   private static final String PHONE = "+97250-0000-000";
   private static final LatLng LOCATION = new LatLng(32.08074, 34.78059);
   private static final float RATING = 4;
   private static final int PRICELEVEL_INT = 2; // used for Places and UserPreferences
   private static final PriceLevel PRICELEVEL = PriceLevel.MODERATE; // used for PlaceDetails
```

Comment:
Note in your PR, but this should be PRICE_LEVEL and not PRICELEVEL.

Reply:
Changed it